static/app/views/issueList/issueViewsHeader.tsx

```diff
@@ -192,6 +195,36 @@ function IssueViewsIssueListHeaderTabsContent({
     queryParams,
   ]);
 
+  const {mutate: updateViews} = useUpdateGroupSearchViews();
+
+  useEffect(() => {
```

Comment:
I'm afraid that this will result in a ton of extra requests. Could you debounce this so that we only make a request every 10 seconds or so?